.travis.yml

```diff
@@ -23,11 +19,10 @@ install:
   - python3 -m pip install -U pip
   - pip3 install --upgrade "git+https://github.com/ibm/detect-secrets.git@master#egg=detect-secrets"
   - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin
-  - go get -u github.com/kardianos/govendor
 
 before_script:
   - bin/detect_secrets.sh
-  - govendor sync
+  - go mod vendor
```

Comment:
Now that this is changing, can you also please update the `README`?

Reply:
I wish there was a jinkx emoticon lol